riscv/csrs.cc

```diff
@@ -13,6 +13,8 @@
 #include "trap.h"
 // For require():
 #include "insn_macros.h"
+// For xs_gatherer_t
```

Comment:
The commit message in ec708aa736c136603af019d65583e06d412f3dfd is misleading. It's not disallowing write to *status -- only the XS field. How about: "Compute *status.XS from extensions instead of being writable"